vllm/engine/llm_engine.py

```diff
@@ -405,6 +408,9 @@ def from_engine_args(
         elif distributed_executor_backend == "mp":
             from vllm.executor.multiproc_gpu_executor import (
                 MultiprocessingGPUExecutor)
+            assert not USE_SPMD_WORKER, (
+                "multiprocessing distributed executor backend does not "
+                "support VLLM_USE_SPMD_WORKER=1")
```

Comment:
I would treat this as a pre-alpha feature, and expect no users but you are developing with this.

You can use `VLLM_RAY_USE_SPMD_WORKER`, and don't need to care about the multiprocessing code path.

Reply:
Changed to `VLLM_USE_RAY_SPMD_WORKER`